client/src/routes/Temple/Session.tsx

```diff
@@ -110,18 +124,20 @@ const Session = () => {
       <Spotlight>
         <TopSafeArea minSize={SPACINGS.TWELVE} />
         {exercise && (
-          <>
-            <Progress
-              index={exercise?.slide.index}
-              length={exercise?.slides.length}
-            />
+          <SpotlightContent>
             <ExerciseSlides
               index={exercise.slide.index}
               current={exercise.slide.current}
               previous={exercise.slide.previous}
               next={exercise.slide.next}
             />
-          </>
+            <ProgressGradient exerciseType={exercise?.slide.current.type}>
```

Comment:
There should be a background gradient behind the progress bar when the facilitator is in the spotlight. It's now covering the mute button, maybe not needed?